packages/env/lib/get-host-user.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+const os = require( 'os' );
+
+/**
+ * Gets information about the host user.
+ *
+ * @return {Object} The host user's name, uid, and gid.
+ */
+module.exports = function getHostUser() {
+	const hostUser = os.userInfo();
+	const uid = ( hostUser.uid === -1 ? 0 : hostUser.uid ).toString();
```

Comment:
Might be worth documenting the windows uid situation inline to avoid future confusion